bigquery/docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       # vm.max_map_count. Note that this file is not used in production
       # deployments; bootstrap checks are still enabled for production
       # deployments.
+      - "ES_JAVA_OPTS=-Xms2g -Xmx2g"
```

Comment:
FYI I [wrote about this](https://discuss.elastic.co/t/import-from-bigquery-to-elasticsearch/155454). If there is interest, we could move/copy the BigQuery -> Elasticsearch to a separate repo.